2015-Fall/andershqst/SwiftLanguage/SwiftLanguage.playground/Contents.swift

```diff
@@ -323,6 +323,38 @@ func myNiceFunc(var arr: Array<Int>) {
 myNiceFunc(arrayStruct)
 arrayStruct[0] // still 1
 
+// The mutating keyword
+// Mutate the properties of a struct from within
+// Replace the struct by assigning to self
+struct MutatingStruct {
+    var x = 0
+    mutating func mutateX(x: Int) {
+        self.x = x
+    }
+    mutating func replaceSelf() {
+        self = MutatingStruct(x: 123)
+    }
+}
+// Must be var
+var ms = MutatingStruct()
+ms.mutateX(42)
+ms.replaceSelf()
+ms.x
```

Comment:
I did not as it shows in the playground. But yes. I guess we can expect people to also look up stuff in the browser